api/admin/routes.py

```diff
@@ -155,32 +155,6 @@ def work_classifications(identifier_type, identifier):
     )
 
 
-@library_route(
```

Comment:
Do only mirrored collections use the urls and methods for cover images?

Reply:
Yes, thats correct. The only time the ability to change the cover image is shown in the admin UI is when the collection has mirror integrations configured.